stackslib/src/net/httpcore.rs

```diff
@@ -855,6 +856,44 @@ struct StacksHttpReplyData {
     stream: StacksHttpRecvStream,
 }
 
+/// Default response handler, for when using StacksHttp to issue arbitrary requests
+#[derive(Clone)]
+struct RPCArbitraryResponseHandler {}
```

Comment:
I think `GenericResponseHandler` would be a better name

Reply:
It's not generic, though.  It's only used when `StacksHttp` is instantiated as a client for HTTP requests that don't have a given type.